Jon-Jonrosario5/Week-3/giffaw/scripts/app.js

```diff
@@ -15,13 +15,11 @@ $(document).on('click','.btn', function(e) {
     method:"GET",
     url:myUrl + search + key,
     success: function(res){
-      console.log(res.data);
-
 
       for (let i=0;i < res.data.length;i++){
```

Comment:
Love the loop logic followed with the append. One way to clean up your code would be to use .forEach()
```js
 res.data.forEach((gif)=>{
      $('.gif-gallery').append(`<img src="${gif.images.downsized.url}">`);
})
```

this just makes it a bit cleaner, but also kinda just style points. Good job.